From cea644fd24dbbf2e2359fd7b6d361a698660d5eb Mon Sep 17 00:00:00 2001
From: Andreas Schneider <asn@samba.org>
Date: Mon, 2 Mar 2015 11:55:01 +0100
Subject: [PATCH] PATCHSET20: doc-xml: Add 'sharesec' reference to 'access
 based share enum'

BUG: https://bugzilla.samba.org/show_bug.cgi?id=11127

Signed-off-by: Andreas Schneider <asn@samba.org>
Reviewed-by: Michael Adam <obnox@samba.org>

Autobuild-User(master): Andreas Schneider <asn@cryptomilk.org>
Autobuild-Date(master): Mon Mar  2 14:33:33 CET 2015 on sn-devel-104

(cherry picked from commit e2ed224653985afa13e906e2a5f3656a18d622c0)
Signed-off-by: Andreas Schneider <asn@cryptomilk.org>
---
 docs-xml/smbdotconf/security/accessbasedshareenum.xml | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

Index: samba-3.6.23/docs-xml/smbdotconf/security/accessbasedshareenum.xml
===================================================================
--- samba-3.6.23.orig/docs-xml/smbdotconf/security/accessbasedshareenum.xml
+++ samba-3.6.23/docs-xml/smbdotconf/security/accessbasedshareenum.xml
@@ -7,7 +7,10 @@
      <para>If this parameter is <constant>yes</constant> for a
     service, then the share hosted by the service will only be visible
     to users who have read or write access to the share during share
-    enumeration (for example net view \\sambaserver).  This has
+    enumeration (for example net view \\sambaserver). The share ACLs
+    which allow or deny the access to the share can be modified using
+    for example the <command moreinfo="none">sharesec</command> command
+    or using the appropriate Windows tools. This has
     parallels to access based enumeration, the main difference being
     that only share permissions are evaluated, and security
     descriptors on files contained on the share are not used in
Index: samba-3.6.23/docs/manpages/smb.conf.5
===================================================================
--- samba-3.6.23.orig/docs/manpages/smb.conf.5
+++ samba-3.6.23/docs/manpages/smb.conf.5
@@ -784,7 +784,9 @@ access based share enum (S)
 .RS 4
 If this parameter is
 \fByes\fR
-for a service, then the share hosted by the service will only be visible to users who have read or write access to the share during share enumeration (for example net view \e\esambaserver)\&. This has parallels to access based enumeration, the main difference being that only share permissions are evaluated, and security descriptors on files contained on the share are not used in computing enumeration access rights\&.
+for a service, then the share hosted by the service will only be visible to users who have read or write access to the share during share enumeration (for example net view \e\esambaserver)\&. The share ACLs which allow or deny the access to the share can be modified using for example the
+sharesec
+command or using the appropriate Windows tools\&. This has parallels to access based enumeration, the main difference being that only share permissions are evaluated, and security descriptors on files contained on the share are not used in computing enumeration access rights\&.
 .sp
 Default:
 \fI\fIaccess based share enum\fR\fR\fI = \fR\fIno\fR\fI \fR
